src/gui/vfs_maccontroller.h

```diff
@@ -0,0 +1,55 @@
+// ================================================================
+// Copyright (C) 2007 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ================================================================
+//
+//  VfsMacController.h
+//
+//
+//  Created by ted on 12/27/07.
+//
+
+
+#ifndef LOOPBACKCONTROLLER_H
+#define LOOPBACKCONTROLLER_H
+
+#include <QtCore>
+
+#include "quotainfo.h"
+#include "accountstate.h"
+
+class VfsMac;
+
+class VfsMacController : public QObject
+{
+    Q_OBJECT
+public:
+    explicit VfsMacController(QString rootPath, QString mountPath, OCC::AccountState *accountState, QObject *parent);
+    //~LoopbackController();
```

Comment:
Wrong Class Name, also who releases and qi_ if there is just a default destructor?

Reply:
Indeed for the class name, and the include guards are wrong as well.

qi_ inherits QObject and is properly parented so it'll be deleted as expected in the QObject dtor, no leakage here.